client/landing/stepper/declarative-flow/domain-transfer.ts

```diff
@@ -35,27 +32,23 @@ const domainTransfer: FlowV1 = {
 				slug: 'domains',
 				asyncComponent: () => import( './internals/steps-repository/domain-transfer-domains' ),
 			},
-			{
-				slug: 'processing',
-				asyncComponent: () => import( './internals/steps-repository/processing-step' ),
-			},
+			STEPS.PROCESSING,
 		];
 	},
 
-	useAssertConditions( navigate ): AssertConditionResult {
+	useSideEffect( _currentStepSlug, navigate ) {
```

Comment:
I just realized this may create infinite redirect and that's why the author opted for `useAssertConditions`. I say let's nuke this whole thing and just wrap the domains and process steps with `stepsWithRequiredLogin`.

Reply:
I'm a bit confused: both `useAssertConditions` and `requiresLoggedInUser` (which is what `stepsWithRequiredLogin` uses under the hood) are both deprecated. Why, and what's the better alternative? Can we make that clearer through better documentation? Seems like the stepper root docs are obsolete.

Reply:
`requiresLoggedInUser` isn't deprecated. 

>  Seems like the stepper root docs are obsolete.

Yes, soon I will be working on a documentation overhaul. Mainly videos and example flows.